src/systems/ec/specs/components/inpage-navigation/demo/data.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
     {
       href: '#inline-nav-2',
-      label: 'Heading 2',
+      label: 'Heading 2 with a long title going on several lines',
```

Comment:
good for validation but I'm not sure that we need to have this label on production, what do you think?